docs/developer/OPERATIONS.md

```diff
@@ -1,41 +1,54 @@
 # OSIDB -- operations
 
-## Code freeze
+## Release
+
+The release process consists of multiple phases and usually takes at least a week.
+Therefore, it needs to be properly planned and communicated. Currently, there is no
+regular release schedule. The release is planned when the team decides so.
+
+Exceptionally, it may happen that a critical issue which blocks the users to perform
+their duties is found. In such case the phases of the release may be shortened as
+needed but due to the reduced space for the testing and communication it requires
+an extreme caution.
```

Comment:
A question, do we want to reference our release strategy here? I know it's not yet finalized but it will one day and it contains some more precise description of the above: https://docs.google.com/document/d/1EaQFw_a4LJ39njwcdZXmdIBegiZo-jjxFQdt72UU9RM/edit#heading=h.8k118o77r4ug

Wdyt?